mne/viz/utils.py

```diff
@@ -67,6 +67,7 @@ def tight_layout(pad=1.2, h_pad=None, w_pad=None, fig=None):
 
     try:  # see https://github.com/matplotlib/matplotlib/issues/2654
         fig.canvas.draw()
+	fig.tight_layout(pad=pad, h_pad=h_pad, w_pad=w_pad)
         fig.set_tight_layout(dict(pad=pad, h_pad=h_pad, w_pad=w_pad))
```

Comment:
the indentation is wrong

also, I assumed we should delete the `set_tight_layout` call, or at least move it into the `else` clause of the `try/except` (assuming e.g. it sets some state to have it redo the tight layout if necessary)